### PR TITLE
chore(splash): fix incorrect es6 syntax on splash page

### LINKED ISSUE #339: Invalid Syntax for Splash Page Snippets
image

I believe this needs to be a `default export` rather than regular export.

### PR DESCRIPTION
**Pull request by TheLarkInn**
Fixes #339

### CODE REVIEW

File: content/index.md [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
 import bar from './foo';
 
-bar():
+bar();
 ```
 
 **foo.js**
 
 ```js
-export function bar() {
```
Comment (asapach) [inline content/index.md]: this named export is syntactically correct

File: content/index.md [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
+default export function bar() {
```
Comment (asapach) [inline content/index.md]: should be either `export default` or left as is

File: content/index.md [hunk 3]
```diff
@@ -5,13 +5,13 @@
 ```js
 import bar from './foo';
```
Comment (sokra) [inline content/index.md]: or you change this to `import { bar } from './foo';?

**Comment by kentcdodds** [comment]
Hey! I was first! #338

**Comment by sokra** [comment]
@kentcdodds :tada:

**Pull request closed without merging**